src/core/lib/iomgr/ev_epoll1_linux.c

```diff
@@ -145,7 +145,7 @@ static const char *kick_state_string(kick_state_type st) {
 }
 
 struct grpc_pollset_worker {
-  kick_state_type kick_state;
+  kick_state state;
```

Comment:
In this case, I feel that is clear that changing this field name to `state` actually increases its value since it makes it more clear that this is the current state of the worker object, not just the state of the kick operation.